Demo/build.gradle

```diff
@@ -59,12 +60,18 @@ android {
 }
 
 dependencies {
-    implementation project(':Card')
-    implementation project(':PayPalDataCollector')
-    implementation project(':PayPalNativeCheckout')
-    implementation project(':PayPalWebCheckout')
-    implementation project(':PayPalUI')
-    implementation project(':Venmo')
+    debugImplementation project(':Card')
+    debugImplementation project(':PayPalDataCollector')
+    debugImplementation project(':PayPalNativeCheckout')
+    debugImplementation project(':PayPalWebCheckout')
+    debugImplementation project(':PayPalUI')
+    debugImplementation project(':Venmo')
+
+    releaseImplementation deps.card
+    releaseImplementation deps.paypalUI
+    releaseImplementation deps.paypalWebCheckout
+    releaseImplementation deps.paypalNativeCheckout
+    releaseImplementation deps.paypalDataCollector
```

Comment:
q: Is there a reason these need to duplicated? I notice there's an extra Venmo dependency between the two but overall these look similar.

Reply:
the idea is that if under Demo you change the build variant to `release` it will pull the `releaseImplementation` dependencies. Which in this case point to the `SNAPSHOT` version. The idea in the future, is to somehow be able to test in the Demo app the `snapshot` versions, the `released` versions and the local ones (local ones are used when you are developing)